docs/marketplace/dev/setup/infrastructure-setup.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Infrastructure setup
+description: This article provides required system infrastructure setup for a Marketplace project.
+template: concept-topic-template
+---
+
+This article provides required system infrastructure setup for a Marketplace project.
```

Comment:
```suggestion
This article provides required system infrastructure requirements for a Spryker Marketplace project to mitigate security risks.
These requirements will be set up for your projects out of the box when you host your Spryker application in SCCOS (spryker cloud - link). In case your system infrastructure (cloud paas or on-premise) is managed , you would need to ensure these requirements are satisfied to ensure proper level of security. 

```

Reply:
done